src/app/components/contact-us-modal/contact-us-modal.component.ts

```diff
@@ -32,22 +34,41 @@ export class ContactUsModalComponent implements OnInit {
 
   constructor(
     public dialog: MatDialog,
+    public http: HttpService,
     @Inject(MAT_DIALOG_DATA) public data: User
   ) { }
 
   ngOnInit(): void {
   }
 
   submit() {
-    this.pending = true;
-// TODO: create mailService
-    setTimeout(() => {
-      this.userName = this.inputName.nativeElement.value;
-      this.userEmail = this.inputEmail.nativeElement.value;
-
-      this.pending = false;
-      this.submited = true;
-    }, 2000);
-  }
+    this.pending = false;
+    this.submited = true;
+    let user = {
+      userName: this.inputName.nativeElement.value,
+      userEmail: this.inputEmail.nativeElement.value,
+      message: this.inputMessage.nativeElement.value,
+    }
+
+    this.http.sendMessage("http://localhost:3002/sendmail", user).subscribe(
+      data => {
+        let res: any = data;
+        console.log(
+          `My name is ${user.userName}. My email is ${user.userEmail}. My message is ${user.message}`
+        );
+      },
+
+      err => {
+        console.log(err);
```

Comment:
Errors must be set to snack bar, not in console.log
We have notification service for this